t/ingredients2.t

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/perl -w
```

Comment:
Why is this t/ingredients2.t? The "2" isn't very descriptive. 🙂

Reply:
It's exactly like ingredients.t, just more test cases. I split it to make it easier / faster to test specific tests or add new test cases.

Reply:
Renamed to ingredients_tags.t , it's a bit different from ingredients.t in fact, as it just looks at the ingredients_original_tags field.